docs/README.md

```diff
@@ -7,11 +7,45 @@ For more details about how OpenWhisk works, see [System overview](./about.md).
 
 ## Setting up the OpenWhisk CLI
 
-You can use the OpenWhisk command line interface (CLI) to set up your namespace and authorization key. 
-Go to [Configure CLI](../README.md#setup-cli) and follow instructions to install it. 
-Note that you must have Python 2.7 installed on your system to use the CLI.
+The OpenWhisk command line interface (CLI) requires Python 2.7.
 
-After OpenWhisk is set up with the CLI, you can begin using it from the command line or through REST APIs.
+- If you cloned the OpenWhisk respository, you will find the CLI in `openwhisk/bin/wsk`.
+
+- Otherwise, download the CLI from an existing deployment.
+You will need to know the base URL for the deployment you want to use and
+install it using [pip](https://pip.pypa.io/).
+
+```
+sudo pip install --upgrade https://{BASE URL}/openwhisk-0.1.0.tar.gz
```

Comment:
Using `docker-machine` this failed for me:

    $ sudo pip install --upgrade https://192.168.99.100/openwhisk-0.1.0.tar.gz
    ...
    SSLError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)

...as I would expect. I could bypass with:

    $ sudo pip install --upgrade https://192.168.99.100/openwhisk-0.1.0.tar.gz --trusted-host 192.168.99.100